database/multistmt/parse.go

```diff
@@ -47,7 +33,7 @@ func Parse(reader io.Reader, delimiter []byte, maxMigrationSize int, replacement
 	// 6. now supports plpgsql trigger bodies
 	var err error = nil
 	// buf is the bytes read from input reader
-	buf := make([]byte, 1024)
+	buf := make([]byte, ParseBufSize)
```

Comment:
nit: we don't need to initialize `buf` here if we do it at the beginning of the loop (right?)

Reply:
agree, though I added the other init/make call to ensure the buffer is clean in each loop iteration, it's sub-optimal performance, I need to do some more testing with the second make call removed.

Reply:
this has been cleaned up, extra init remove